### ISSUE TITLE
SteamUserStats.GetNumAchievements () always return 0

### ISSUE THREAD

**Issue by fabiobh** [rlabrecque/Steamworks.NET#211]
I use the following code on start method:

```
CGameID m_GameID;
if (SteamManager.Initialized) {
			string name = SteamFriends.GetPersonaName ();
			Debug.Log (name+SteamUser.GetSteamID() );

			// Cache the GameID for use in the Callbacks
			m_GameID = new CGameID (SteamUtils.GetAppID ());

			Debug.Log ("gameID: " + m_GameID);
			Debug.Log ("achievement: " + SteamUserStats.GetNumAchievements ());

		} else {
			Debug.Log ("Steam not initialized");
		}
```
The appID, username steamID from Steam is showing correctly, I use this code for all my steam games that have achievements.

But now all my games are showing 0 achievements when I test them in Unity Editor, this was not happen some time ago.

**Comment by rlabrecque** [comment]
Did you call SteamUserStats.RequestCurrentStats() before, and wait for the result before calling this code?

**Comment by fabiobh** [comment]
@rlabrecque I didn't call SteamUserStats.RequestCurrentStats().
I don't know that is necessary to call SteamUserStats.RequestCurrentStats() before work with achievements.
I didn't call this method in the first time that I use Steamworks.NET and I got achievements even without call SteamUserStats.RequestCurrentStats().

But I use SteamUserStats.RequestCurrentStats now and I got the achievement numbers now, It works, thank you.
